tools/deviceGen/output/traits_defaults.hlsl

```diff
@@ -0,0 +1,382 @@
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct4x8BitPackedSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct4x8BitPackedMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct16BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct16BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct16BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct32BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct32BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct32BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct64BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct64BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProduct64BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating8BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating8BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating8BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating16BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating16BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating32BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating32BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating64BitUnsignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating64BitSignedAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated, bool, false);
+NBL_GENERATE_GET_OR_DEFAULT(storageTexelBufferOffsetAlignmentBytes, size_t, std::numeric_limits<size_t>::max());
+NBL_GENERATE_GET_OR_DEFAULT(uniformTexelBufferOffsetAlignmentBytes, size_t, std::numeric_limits<size_t>::max());
+NBL_GENERATE_GET_OR_DEFAULT(maxBufferSize, size_t, MinMaxSSBOSize);
```

Comment:
`size_t` isn;t a thing in HLSL https://godbolt.org/z/8nnfnToqs

use `uint64_t` instead

Reply:
Done